src/main/java/org/tron/core/db/Manager.java

```diff
@@ -222,13 +224,20 @@ public ProposalStore getProposalStore() {
     return this.popedTransactions;
   }
 
+  public BlockingQueue<TransactionCapsule> getRepushTransactions() {
+    return repushTransactions;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Return a copy of "repushTransactions". ([squid:S2384](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2384))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1298&issues=AWUzb7VyW2AOP0CkArPs&open=AWUzb7VyW2AOP0CkArPs)

[ISSUE_KEY]: AWUzb7VyW2AOP0CkArPs